GOVERNANCE.md

```diff
@@ -0,0 +1,32 @@
+# CNI Governance
+
+This document defines project governance for the project.
+
+## Voting
+
+The CNI project employs "organization voting" to ensure no single organization can dominate the project.
+
+Individuals not associated with or employed by a company or organization are allowed one organization vote.
+Each company or organization (regardless of the number of maintainers associated with or employed by that company/organization) receives one organization vote.
+
+In other words, if two maintainers are employed by CoreOS, two by WeaveWorks, two by Red Hat, and one maintainer is an un-affiliated individual, a total of four "organization votes" are possible; one for CoreOS, one for WeaveWorks, one for Red Hat, and one for the un-affiliated individual.
```

Comment:
No love for Tigera or Pivotal 😄  ?  

perhaps we should say "are employed by Company X, two by Company Y..."